packages/common/tests/repo.test.ts

```diff
@@ -1,127 +1,71 @@
 import * as auth from '@adxp/auth'
 
-import Repo from '../src/repo/index'
-import IpldStore from '../src/blockstore/ipld-store'
-
+import { Repo } from '../src/repo'
+import { MemoryBlockstore } from '../src/blockstore'
 import * as util from './_util'
-import TID from '../src/repo/tid'
 
 describe('Repo', () => {
-  let ipldStore: IpldStore
+  let blockstore: MemoryBlockstore
   let authStore: auth.AuthStore
   let repo: Repo
-  const namespaceId = 'did:example:test'
-  const otherNamespace = 'did:example:other'
+  let repoData: util.RepoData
 
   it('creates repo', async () => {
-    ipldStore = IpldStore.createInMemory()
+    blockstore = new MemoryBlockstore()
     authStore = await auth.MemoryStore.load()
     await authStore.claimFull()
-    repo = await Repo.create(ipldStore, await authStore.did(), authStore)
+    repo = await Repo.create(blockstore, await authStore.did(), authStore)
   })
 
-  it('adds a valid signature to commit', async () => {
-    const commit = await repo.getCommit()
-    const verified = await auth.didPlugins.verifySignature(
-      repo.did,
-      commit.root.bytes,
-      commit.sig,
-    )
-    expect(verified).toBeTruthy()
-  })
+  it('does basic operations', async () => {
+    const collection = repo.getCollection('bsky/posts')
```

Comment:
this test has basic usage of a repo.

You get a collection & then do operations on that collection. These then call back to the repo to do a safe commit